pkg/repository/container_redis.go

```diff
@@ -306,3 +311,97 @@ func (cr *ContainerRedisRepository) GetFailedContainerCountByStubId(stubId strin
 
 	return failedCount, nil
 }
+
+func (c *ContainerRedisRepository) GetConcurrencyLimitByWorkspaceId(workspaceId string) (*types.ConcurrencyLimit, error) {
```

Comment:
should we have a workspace repo instead of putting this at the container level?

Reply:
I actually have the workspace repo in the Temporary auth token PR